javaslang/src/main/java/javaslang/Value.java

```diff
@@ -516,7 +516,7 @@ default CharSeq toCharSeq() {
      */
     @SuppressWarnings("unchecked")
     @GwtIncompatible("reflection is not supported")
-    default T[] toJavaArray(Class<T> componentType) {
+    default T[] toJavaArray(Class<? super T> componentType) {
```

Comment:
This (and the previous) also break for primitives, e.g.
```java
final Value<Integer> value = of(1, 2, 3);
final Integer[] ints = value.toJavaArray(int.class);
```

I guess this is why `Java` provides an actual array instead...